l10n_it_fatturapa_export_zip/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_wizard_fatturapa_export_manager,access_wizard_fatturapa_export_manager,model_wizard_fatturapa_export,account.group_account_invoice,1,1,1,1
```

Comment:
Queste 2 ACL mi sembrano ridondanti a fronte di:

_Access rights are additive, a user’s accesses are the union of the accesses they get through all their groups e.g. given a user who is part of group A granting read and create access and a group B granting update access, the user will have all three of create, read, and update._

(rif. https://www.odoo.com/documentation/14.0/developer/reference/addons/security.html#access-rights)

Inoltre il permesso `perm_unlink` può essere impostato a `0` così come fatto da Odoo in https://github.com/odoo/odoo/commit/65530dfd6a0486f44134917f5cc42a2c30b95762#diff-1091362b90b9a9be15980cbc00427e48d5f2f86574ad10e9ebeb87a7c6a9f6b7

Pertanto inserirei solo:
`access_wizard_fatturapa_export_user,access_wizard_fatturapa_export_user,model_wizard_fatturapa_export,account.group_account_invoice,1,1,1,0`